docs/hardware.md

```diff
@@ -41,6 +41,13 @@ These boards run MakeCode Arcade games. Choose a board to find out more about it
         "url": "https://www.kitronik.co.uk/arcade",
         "variant": "hw---samd51"
     },
+    {
+        "name": "Ovobot Xtron",
+        "description": "A programmable microcomputer that can be used for making MakeCode Arcade games, controlling Scratch sprites and creating works from electronics to robots.",
```

Comment:
this is too long, it will bleed the card.

Reply:
I have shortened it.